### DISCUSSION TITLE
Achievements not getting updated

### DISCUSSION THREAD

**Discussion by 1-ARIjitS** [community/community#41755] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My Github Achievements are not getting updated. Please fix this bug. I have made several PRs and must have got many pull shark achievements but its showing only one. I don't have any other badges also, but i must be getting quickdraw and other badges. Please fix this bug ASPA

**Comment by radar-blip** [comment]
You dont get a new achievement badge everytime you make a PR.
You get the Pull Shark acheivement after making 2 PRs
You will get the next level (bronze) at 16 PRs

See link below for better understanding:
[Achievement-Badges](https://github.com/drknzz/GitHub-Achievements)

Here are different levels of Pull Shark:

![pully](https://user-images.githubusercontent.com/39196767/207747683-8c8d1375-63db-4d3f-9e0b-8d784927afcd.png)

**Comment by Manishak798** [comment]
How people add x2 with their achievements

**Reply by radar-blip** [reply]
[Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by RamtinKosari** [reply]
Hi @Manishak798 
Use this [guide](https://github.com/4xmen/Get-Github-Achievements/tree/main) to earch achievements like x2

**Comment by RamtinKosari** [comment]
Hi Dear @1-ARIjitS 

Double-check your contributions: Make sure that your pull requests (PRs) have been successfully merged into the target branch. Only merged PRs count towards achievements and badges.

Verify the activity period: Some achievements and badges are awarded for contributions within a specific timeframe. Check if your contributions fall within the required period.

Check your profile settings: Ensure that your GitHub profile visibility settings allow others to see your contributions and achievements. Go to your GitHub profile settings and verify if the "Contribution status" is set to "Public".

Wait for updates: Occasionally, there may be delays in updating achievements and badges on GitHub. Give it some time, as it can take a while for the system to update and reflect your recent contributions accurately.

and the last thing you should know is that gaining PRs multi Achievements Requires 1, 16, 128, and 1024 PRs in order to gain 1, x2, x3 and x4 PRs

**Reply by BBDBABDBU** [reply]
I had this problem but as you mentioned i checked my contributions and fixed that thank you

**Reply by RamtinKosari** [reply]
your welcome

**Comment by smaaliyan** [comment]
It's possible that there might be a delay in updating your GitHub achievements and badges. Sometimes, it takes a bit of time for the system to catch up with your contributions.

Here's what you can do:

1) Wait a Bit:  Give it some time. GitHub achievements and badges might not update instantly. They could show up after a while.

2) Check PRs:  Double check that your pull requests meet the criteria for the achievements you're expecting. Make sure they're merged or closed.

3) Contact GitHub Support:  If the issue persists, consider reaching out to GitHub Support. They can look into any potential technical issues causing the delay.

Remember, sometimes it's just a matter of waiting, and your achievements should update eventually.

**Comment by rromulos** [comment]
I'm not sure, but there is an option to "Include private contributions on my profile". Maybe it has something to do with it.

**Comment by meemknight** [comment]
Guys, this is still not fixed :))) 
![image](https://github.com/user-attachments/assets/f5407a57-506c-4e63-b7ba-0625139bd74b)

I have a repo with 100 stars, why is that one not getting picked?

**Reply by alihesari** [reply]
same issue for me

**Reply by rawcache** [reply]
128 stars is the next one, the bronze badge. After that its 512 and 4096 stars for the silver and gold, respectively.

**Reply by alihesari** [reply]
> 128 stars is the next one, the bronze badge. After that its 512 and 4096 stars for the silver and gold, respectively.

I could not find documentation about this, do you have any link?

**Comment by tuyennn** [comment]
+1 for the issue not fixed
